languages/elixir/exercises/concept/booleans/test/rules_test.exs

```diff
@@ -0,0 +1,57 @@
+defmodule RulesTest do
+  use ExUnit.Case
+
+  test "if ghost gets eaten" do
```

Comment:
```suggestion
  # @tag :pending
  test "if ghost gets eaten" do
```
Minor point, but I like to leave a commented out pending tag on the first as a clue that the line can be commented, then leave the rest of the lines as `     @tag :pending`.
Would it make more sense to specify that the student can comment or remove the lines in a document instead of as comments in the test file?

Reply:
Sure, but I think in the basics exercise this mechanic is explained.

Reply:
Also, in the browser, the student never sees the test file, as I understand it